tools/scripts/build/mobile-curriculum.test.ts

```diff
@@ -1,10 +1,14 @@
-const path = require('path');
-const fs = require('fs');
-const { AssertionError } = require('chai');
-const envData = require('../../../config/env.json');
-const { mobileSchemaValidator } = require('./mobileSchema');
+import path from 'path';
+import fs from 'fs';
+import { AssertionError } from 'chai';
+import envData from '../../../config/env.json';
+import { SuperBlocks } from '../../../config/certification-settings';
+import { mobileSchemaValidator } from './mobileSchema';
+import { superBlockMobileAppOrder } from './build-external-curricula-data';
 
 if (envData.clientLocale == 'english' && !envData.showUpcomingChanges) {
+  const ver = 'v1.0.0';
```

Comment:
```suggestion
  const VERSION = 'v1.0.0';
```
It's a constant and abbrvs are confusing!